dbt_meshify/dbt_projects.py

```diff
@@ -121,6 +148,13 @@ def get_manifest_node(self, unique_id: str) -> ManifestNode:
         """Returns the catalog entry for a model in the dbt project's catalog"""
         return self.manifest.nodes.get(unique_id, {})
 
+    def select_resources(
```

Comment:
Is this necessary to include in this class? i see it's overridden in the inherited class, i'm just curious about the utility to also include it here.

Reply:
Nah, this can go in the bin.